.pre-commit-config.yaml

```
@@ -444,6 +444,17 @@ repos:
         require_serial: true
         files: ^providers/src/airflow/providers/.*\.py$
         additional_dependencies: ['rich>=12.4.4']
+      - id: check-breeze-uv-lock-consistent
+        name: Check breeze uv.lock
+        description: Breeze uv.lock should be in sync with pyproject.toml
+        language: python
+        entry: uv lock
+        args:
+          - --project
+          - dev/breeze
+        pass_filenames: false
+        require_serial: true
+        additional_dependencies: ['uv']
```

```suggestion
        additional_dependencies: ['uv==0.5.2']
```

We are lockling `uv` to specific version, because they change and release so often thet they might (and did) break things often. We do not want `uv` breakage to break our CI, that's why we "lock" it. And we also have a separate `pre-commit` that we run manually periodically to upgrade UV automatically - and it should be also added there. It is `update-installers` pre-commit.